lib/api/widget/functions.php

```diff
@@ -306,6 +308,9 @@ function beans_get_widget( $needle = false ) {
  * @return string Content with shortcodes filtered out.
  */
 function beans_widget_shortcodes( $content ) {
+	if ( ! isset( $GLOBALS['_beans_widget'] ) ) {
```

Comment:
add a new line above each control.

Reply:
Doh! Seems like there ought to be a code sniff rule for that. 🐽